styles.css

```diff
@@ -98,12 +104,16 @@ header img {
 
```

Comment:
Unrelated to this pull req, but `text-align` on line 102 should be either `center` or `justify`